astropy/units/quantity_helper/function_helpers.py

```diff
@@ -140,6 +140,8 @@
     np.pv, np.rate}
 if NUMPY_LT_1_18:
     IGNORED_FUNCTIONS |= {np.rank}
+else:
+    IGNORED_FUNCTIONS |= {np.alen}
```

Comment:
Ha, was just about to comment on this - saw I needed refresh - and voila, it is OK!